lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -51,6 +51,11 @@
         description:
             - Name of the newly deployed guest
         required: True
+   new_name:
+        description:
+            - Rename a VM
+            - New Name of the exising VM
+        required: False
```

Comment:
Please add the following line, at the same indentation level as `required: False`
`version_added: "2.3`